test/unit/postprocessing/test_plots.py

```diff
@@ -1,8 +1,13 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-import matplotlib
-matplotlib.use('TkAgg')
+# On MacOS we need to use TKAgg before importing matplotlib.pyplot.
+# This used to work on all platforms until matplotlib 3.2.0 broke it on Linux.
+# Consider removing the OS-based if after a future release of matplotlib.
```

Comment:
Can you file as issue and link in the comment. I'm really not a fan of code like this.

Reply:
done #320